src/index.js

```diff
@@ -129,22 +129,29 @@ function updateState(state, history) {
 }
 // /updateState
 
+// createHistory
+function createHistory(state) {
```

Comment:
Not so sure about that since there is already `initialHistory` term used and is more about _creating_ fresh history than initializing anything.